### ISSUE TITLE
kubecm completion bash adds unrelated strings to output

### ISSUE THREAD

**Issue by jakshi** [sunny0826/kubecm#490]
When you execute: `kubecm completion bash`
it adds "A new release..." unrelated strings to the end of bash completion output:

```
...
if [[ $(type -t compopt) = "builtin" ]]; then
    complete -o default -F __start_kubecm kubecm
else
    complete -o default -o nospace -F __start_kubecm kubecm
fi

# ex: ts=4 sw=4 et filetype=sh


A new release of kubecm is available: dev → 0.19.1
To upgrade, run: brew update && brew upgrade kubecm
https://github.com/sunny0826/kubecm/releases/tag/v0.19.1
```

if you use `brew install kubecm`, then brew uses output of kubecm completion bash to generate bash completion script.

So you got:

```
A new release of kubecm is available: dev → 0.19.1
To upgrade, run: brew update && brew upgrade kubecm
https://github.com/sunny0826/kubecm/releases/tag/v0.19.1
```

into end of your bash completion script for kubecm.

So when you start your terminal you see something like:

```
-bash: $'\E[0': command not foundug 2022 at 11:29
-bash: 33mA: command not found
-bash: $'36mdev\E[0m': command not found
-bash: $'32m0.19.1\E[0m': command not found
-bash: $'\E[0': command not found
-bash: 33mhttps://github.com/sunny0826/kubecm/releases/tag/v0.19.1: No such file or directory
```

because bash completion tries to execute:

```
[0;33mA new release of kubecm is available:[0m [0;36mdev[0m ‚Üí [0;32m0.19.1[0m
[0;33mhttps://github.com/sunny0826/kubecm/releases/tag/v0.19.1[0m
```

in the end of kubecm bash completion script.

desired behavior - no extra lines unrelated to bash completion script added to the output of `kubecm completion bash` output.

**Comment by sunny0826** [comment]
Thank you very much for your feedback! This is a bug caused by a new feature, I will fix it as soon as possible.

**Comment by sunny0826** [comment]
This issue has been fixed in [v0.19.2](https://github.com/sunny0826/kubecm/releases/tag/v0.19.3), if the problem persists, please reopen this issue.
